Mage/src/main/java/mage/abilities/effects/common/DamagePlayersEffect.java

```diff
@@ -38,6 +38,10 @@ public DamagePlayersEffect(Outcome outcome, DynamicValue amount) {
         this(outcome, amount, TargetController.ANY);
     }
 
+    public DamagePlayersEffect(DynamicValue amount, TargetController controller) {
+        this(Outcome.Damage, amount, controller, "{this}");
+    }
+
     public DamagePlayersEffect(Outcome outcome, DynamicValue amount, TargetController controller) {
```

Comment:
(Separate scope for sure, but I don't see the point in putting Outcome in any of these constructors)